src/arrayOfPeople.js

```diff
@@ -39,6 +39,15 @@
  */
 function arrayOfPeople(people) {
   // write code here
+  let peopleInRoom = [];
```

Comment:
[prefer-const](https://mate-academy.github.io/style-guides/javascript.html#references--prefer-const)
But I miss it the first time, approve